README.md

```diff
@@ -1,5 +1,6 @@
 # Manacher
 
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
```

Comment:
Can you move this badge to the last after the license badge?
Also, make the style uniform. Use the default plastic style instead of flat square style.

Reply:
updated, please check